.github/workflows/test-release.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         working-directory: ./release-notes-fetcher
         # The var CAMUNDA_RELEASE_NAME should be the same as GITHUB_REF_NAME but it's hard-coded as 8.3.1 tag
         # and we just want to see if a valid version can fetch notes
-        run: set -o pipefail; CAMUNDA_RELEASE_NAME=8.3.1 ./release-notes-fetcher | tee release_notes.txt
+        run: set -o pipefail; CAMUNDA_RELEASE_NAME=8.5.1 ./release-notes-fetcher | tee release_notes.txt
```

Comment:
@jessesimpson36, do we need to update this for every new minor, or every new release?

Reply:
We do not. only when a change happens that changes where we have to look for the underlying github tags / where the release notes are stored. 

In 8.3.1, the release notes are in the camunda/zeebe repo.  If we release an 8.3.10 (or some other new patch version), I think the release-notes-fetcher from the `stable/8.3` branch will run, which will pull the information from the `camunda/zeebe` repo.  

Does that align with what you'd expect?

Reply:
yes, it was more a question to check if we need to change something here: https://confluence.camunda.com/display/HAN/Camunda+Platform+8+SaaS+-+Rollout